Dockerfile

```diff
@@ -147,8 +147,9 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT 8a87001d09852058f08a807ab6e8491d57ca1e88
-RUN git clone https://github.com/docker/docker-py.git /docker-py \
+#ENV DOCKER_PY_COMMIT 8a87001d09852058f08a807ab6e8491d57ca1e88
```

Comment:
Yup, it will go. As I put above, it needs an update here to point to the right docker-py once the changes are available there. Otherwise this PR is reviewable as it stands. Just cannot be merged until that final update.